### PR TITLE
PE-4430: makes a wording to use a placeholder

### PR DESCRIPTION
**Pull request by matibat**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/52rdhhtcpp4t0

### CODE REVIEW

**Comment by matibat** [comment]
TODO: update the texts in Localizely

File: lib/turbo/topup/views/topup_payment_form.dart [hunk 1]
```diff
@@ -613,14 +614,24 @@ class TimerWidget extends StatefulWidget {
   final int durationInSeconds;
   final VoidCallback onFinished;
   final TextStyle? textStyle;
-  final Widget Function(BuildContext context, int secondsLeft)? builder;
+  final bool isFetching;
+  final bool hasError;
+  final bool humanReadable;
+  final bool humanReadableWithPadding;
+  final bool boldTimer;
+  // final Widget Function(BuildContext context, int secondsLeft)? builder;
```
Comment (matibat) [inline lib/turbo/topup/views/topup_payment_form.dart:622]: Remove unnecessary

File: lib/turbo/topup/views/topup_payment_form.dart [hunk 2]
```diff
@@ -613,14 +614,24 @@ class TimerWidget extends StatefulWidget {
...
     required this.durationInSeconds,
     required this.onFinished,
     this.textStyle,
-    this.builder,
+    this.isFetching = false,
+    this.hasError = false,
+    required this.humanReadable,
+    this.humanReadableWithPadding = false,
+    this.boldTimer = false,
+    // this.builder,
```
Comment (matibat) [inline lib/turbo/topup/views/topup_payment_form.dart:634]: Remove unnecessary

File: lib/turbo/topup/views/topup_payment_form.dart [hunk 3]
```diff
@@ -866,19 +943,21 @@ class QuoteRefreshWidgetState extends State<QuoteRefreshWidget> {
...
+                    //   child: Text(
+                    //     appLocalizationsOf(context).quoteUpdatesIn,
+                    //     style: ArDriveTypography.body.captionBold(
+                    //       color: ArDriveTheme.of(context)
+                    //           .themeData
+                    //           .colors
+                    //           .themeFgDefault,
+                    //     ),
+                    //   ),
+                    // ),
```
Comment (matibat) [inline lib/turbo/topup/views/topup_payment_form.dart:957]: Remove unnecessary

File: lib/turbo/topup/views/topup_payment_form.dart [hunk 4]
```diff
@@ -1006,16 +1085,17 @@ class QuoteRefreshWidgetState extends State<QuoteRefreshWidget> {
...
-                    ),
+                    // Text(
+                    //   appLocalizationsOf(context).quoteUpdatesIn,
+                    //   style: ArDriveTypography.body.captionBold(
+                    //     color: ArDriveTheme.of(context)
+                    //         .themeData
+                    //         .colors
+                    //         .themeFgDefault,
+                    //   ),
+                    // ),
```
Comment (matibat) [inline lib/turbo/topup/views/topup_payment_form.dart:1096]: Remove unnecessary

**Review by matibat — COMMENTED** [review]
Let's cleanup these

File: lib/turbo/topup/views/topup_payment_form.dart (hunk 1 above)
Comment (matibat) [reply lib/turbo/topup/views/topup_payment_form.dart]: 808822d6358c8ae09a345c4a6f902aef29fbd95e

File: lib/turbo/topup/views/topup_payment_form.dart (hunk 2 above)
Comment (matibat) [reply lib/turbo/topup/views/topup_payment_form.dart]: 808822d6358c8ae09a345c4a6f902aef29fbd95e

File: lib/turbo/topup/views/topup_payment_form.dart (hunk 3 above)
Comment (matibat) [reply lib/turbo/topup/views/topup_payment_form.dart]: 808822d6358c8ae09a345c4a6f902aef29fbd95e

File: lib/turbo/topup/views/topup_payment_form.dart (hunk 4 above)
Comment (matibat) [reply lib/turbo/topup/views/topup_payment_form.dart]: 808822d6358c8ae09a345c4a6f902aef29fbd95e

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM

**Pull request merged**